packages/storybook/src/stories/Components/BasicModal/Modal.stories.tsx

```diff
@@ -15,20 +19,37 @@ interface MenuItem {
 
 interface ModalStoryProps {
   menuItems: MenuItem[];
-  width?: number;
-  height?: number;
+  width?: Container | number;
```

Comment:
what is exactly this "Container" enum?

Reply:
In case in the future you want to have different default heights and widths we gonna use the container enum, or pass this props directly that's why is type Container | number;